icgcget/clients/pdc/pdc_client.py

```diff
@@ -50,7 +57,15 @@ def access_check(self, key, data_paths=None, path=None, repo=None, output=None,
         env_dict['AWS_ACCESS_KEY_ID'] = key
         env_dict['AWS_SECRET_ACCESS_KEY'] = secret_key
         for data_path in data_paths:
-            call_args = [path, 's3', self.url, 'cp', data_path, output + '/', '--dryrun']
+            call_args = []
+            if self.docker:
+                call_args = ['docker', 'run', '-e', 'AWS_ACCESS_KEY_ID=' + key, '-e', 'AWS_SECRET_ACCESS_KEY=' +
+                             secret_key, '-t', '-v', output + ':/icgc/mnt', '--rm', 'icgc/icgc-get:0.3.1']
```

Comment:
Pull out the version of this and centralize. We want to make such things easier to change in the future. We need to document the build and release process as well.